src/fs/storage.ts

```diff
@@ -225,12 +226,16 @@ export class SplitStrategyStorage implements FSStorage {
   }
 }
 
-const copy = async (from: FSStorage, to: FSStorage) => {
+const initializeFromStorage = async (from: FSStorage, to: FSStorage) => {
   const files = await from.ls();
-  return Promise.all(
-    files.map(async (f) => {
+  const projectName = await from.projectName();
+  if (projectName) {
+    await to.setProjectName(projectName);
+  }
+  return Promise.all([
+    ...files.map(async (f) => {
```

Comment:
Oops, this is also odd, from when I updated project name unconditionally and so put it inside the Promise.all. The extra list is now pointless.